docs/agile-with-CEA/activities.rst

```diff
@@ -2,37 +2,37 @@ Activities
 ===========
 
 This is a guide of Activities carried out during the development of CEA. This activities are part of our concept of operation
-udner responsability of our the Scrum Master
+under responsibilities of our the Scrum Master
 
 Planning event
 ---------------
 
-Duration: 1 - 4 hours.
-Frequency: After every block of work (sprint) (max of 1 month)
-Scope: Define what user stories should go in the next block of work (sprint). These stories are attached to a new time-bounded milestone
+* Duration: 1 - 4 hours.
+* Frequency: After every block of work (sprint) (max of 1 month)
+* Scope: Define what user stories should go in the next block of work (sprint). These stories are attached to a new time-bounded milestone
 Attendees: Scrum Master, Product Owner, Product Sponsor, and Development Team.
 
 Daily stand-up
 --------------
 
-Duration: 15 min.
-Frequency: daily
-Scope: Provide early support and mentoring
+* Duration: 15 min.
+* Frequency: daily
+* Scope: Provide early support and mentoring
 Attendees: Scrum Master and Development Team.
 
 Review event
 -------------
 
-Duration: 1 - 4 hours.
-Frequency: at the end of the sprint
-Scope: Present a realtime DEMO and receive feedback of the Product Sponsor.
+* Duration: 1 - 4 hours.
+* Frequency: at the end of the sprint
+* Scope: Present a realtime DEMO and receive feedback of the Product Sponsor.
```

Comment:
feedback of the product owner and customer team mainly. The product owner is supposed to be on top of 1. what is technically possible, what users need, what the priorities are, and what should be accepted and what not. The customer team can also give feedback if they are there.

Reply:
i think we can replace this one with the SM & PO meeting that happens one day before the Planning event.
there, we discuss what goes into the product backlog, set priorities, and discuss the users need. 

and the original description is what we will do in the Roadmap Planning and Review Event.